Servers/Administration/OJS.Servers.Administration/Controllers/ContestCategoriesController.cs

```diff
@@ -85,9 +85,15 @@ protected override Task BeforeEntitySaveOnCreateAsync(ContestCategory entity, Ad
     protected override Task BeforeEntitySaveOnDeleteAsync(
         ContestCategory entity,
         AdminActionContext actionContext)
-        => Task.WhenAll(
+    {
+        entity.Children = this.contestCategoriesData
+            .GetByIdQuery(entity.Id)
+            .SelectMany(c => c.Children).ToList();
```

Comment:
Place .ToList() below .SelectMany() for better readability.

Reply:
It's fixed.